src/script/components/InputBar/InputBarEditor/RichTextEditor/plugins/BlockquotePlugin/BlockquotePlugin.tsx

```diff
@@ -129,6 +129,7 @@ const registerBlockquoteBackspaceCommand = (editor: LexicalEditor) => {
       editor.update(() => {
         lastChild.remove();
         editor.dispatchCommand(INSERT_PARAGRAPH_COMMAND, undefined);
+        editor.dispatchCommand(INSERT_PARAGRAPH_COMMAND, undefined);
```

Comment:
adding an extra new line on backspace actually escape the blockquotes, and the added line is consistent with the message list md rendering